Tests/StringHelperTest.php

```diff
@@ -248,11 +248,11 @@ public function seedTestStrrev(): \Generator
     public function seedTestStrspn(): \Generator
     {
         yield ['A321 Main Street', '0123456789', 1, 2, 2];
-        yield ['321 Main Street', '0123456789', null, 2, 2];
+        yield ['321 Main Street', '0123456789', null, 2, 3];
```

Comment:
Why does the expected result change here?

Reply:
I reverted the change, since while this throws a deprecation notice in PHP 8.1+, it doesn't throw an error.